content/concepts/basics/05-rest-api.md

```diff
@@ -3,3 +3,35 @@ title: "REST API"
 metaTitle: "sensenet - REST API"
 metaDescription: "sensenet REST API"
 ---
+
+sensenet offers you a rich API allowing you to manage, integrate and deliver content on all types of clients, devices and platforms. This section explains the API structure and main concepts in more depth.
+
+sensenet content repository is an [OData](https://www.odata.org/) producer which means your application can consume our [OData](https://www.odata.org/) service to create web and mobile apps or any other type of content based applications. It supports CRUD operations, properties, binary streams, paging options, custom queries and many more.
+
+The requested resource can be any content in the repository (since everything is a content in sensenet, it could be really anything) that is permitted for the current user.
+
+[dig deeper into sensenet REST API](/api-docs/basic-concepts)
+
+## Content delivery
+
+The requested resource can be any content in the repository that is permitted for the current user. Since everything is a content in sensenet this means that every item (user, folder, document, etc) can be accessed through the same API.
+
+Resources can be accessed as individual entities or as collections (containers) of content. Bases on the way of accessing the result will contain a content's or its children's metadata. The reponse is always in json format which you can easily process and built into your own application.
```

Comment:
```suggestion
Resources can be accessed as individual entities or as collections (containers) of content. Bases on the way of accessing the result will contain a content's or its children's metadata. The response is always in json format which you can easily process and build into your own application.
```

Reply:
Bases on the way of accessing the result will contain a content's or its children's metadata.

I don't understand this sentence, could you shed some light on it please.